### PR TITLE
Made basic UI for game page

### PR DESCRIPTION
**Pull request by xmasterpiecex**

### CODE REVIEW

**Comment by uspiritwolf** [comment]
I think it is necessary to structure components by folder, that is, all "Game Scene Component", in the folde, like "GameScene"

File: src/App.tsx [hunk 1]
```diff
@@ -5,20 +5,19 @@ import CssBaseline from "@mui/joy/CssBaseline";
...
-import NotificationCenter from '@app/features/notifications/NotificationCenter';
+import DefaultBackground from "@app/components/Background";
+import ThemeToggle from "@app/components/ThemeToggle";
+import { useEffect } from "react";
 
-import { useEffect } from 'react';
-
-import RoomRPC from '@app/core/GameRPC';
+import RoomRPC from "@app/core/GameRPC";
+import NotificationCenter from "./features/notifications/NotificationCenter";
```
Comment (uspiritwolf) [inline src/App.tsx]: Please use absolute path instead relative (i.e. @app)

File: src/components/DrawerMain.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  DialogTitle,
+  IconButton,
+  ModalClose,
+  Sheet,
+} from "@mui/joy";
+import Drawer from "@mui/joy/Drawer";
+import Menu from "@mui/icons-material/Menu";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import SoundSlide from "./SoundSlide";
```
Comment (uspiritwolf) [inline src/components/DrawerMain.tsx]: The same. Please use absolute path instead relative (i.e. @app)

**Review by uspiritwolf — CHANGES_REQUESTED** [review]

File: src/components/DrawerMain.tsx (hunk 2 above)
Comment (xmasterpiecex) [reply src/components/DrawerMain.tsx]: Got it!

**Review by uspiritwolf — APPROVED** [review]

**Pull request merged**